papertty/drivers/drivers_full.py

```diff
@@ -270,6 +270,177 @@ def sleep(self):
         self.send_data(0xa5)
 
 
+class EPD3in7(WaveshareFull):
+    """Waveshare 3.7" - monochrome"""
+
+    def __init__(self):
+        super().__init__(name='3.7" BW', width=280, height=480)
+
+    # not ported: 4-greyscale mode, partial refresh (~600-900 ms, EPD_3in7.c)
```

Comment:
It's unclear which file is referenced - assuming related to Waveshare sample code. Could be bit more verbose here (and reference the Python source instead):
```suggestion
    # The 3.7" display hardware supports B/W and 4-shade grayscale modes, but the
    # grayscale mode is not yet implemented here. Reference code in Waveshare's repo:
    # https://github.com/waveshare/e-Paper/blob/master/RaspberryPi_JetsonNano/python/lib/waveshare_epd/epd3in7.py
    # Updating the screen in grayscale mode takes approximately 600-900 ms.
```